shared/settings/nav/index.native.tsx

```diff
@@ -54,7 +54,7 @@ function SettingsNav(props: Props) {
             {
               badgeNumber: props.badgeNumbers[TabConstants.settingsTab],
               onClick: () => props.onTabChange(Constants.accountTab),
-              text: 'Account',
+              text: 'Your account',
```

Comment:
Can you capitalize `Your Account` to be consistent with the other stuff in this menu?

Reply:
Should `Screen Protector` below be `Screen protector` then? That's what made me think they should be capitalized.